index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="assets/css/style.css">
+   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+    integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+</head>
+
+<!--Body-->
+<body>
+
+  <!--nav-->
+
+  <header>
+    <nav>
+      <h1>my logo</h1>
+   <img src="assets/images/Union.svg" alt="">
+    </nav>
+  </header>
+
+  <!--Hero section-->
+
+  <section class="hero">
+    <div>
+      <h1>I’m Afimia Fidelis. <br>
+Glad to see <br> you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] Kindly use any `hx` except the `h1` tag here instead of the `anchor` tag because it is just a heading or I would say sub heading. As I mentioned earlier that it is important to have correct semantics in your portfolio to make it more acccessible.

**Tip:** *Please always take user point of view while development of the website or project which will help you to understand about what you can provide to the user.*